libs/components/behavioral/src/lib/tissue-library-behavior/tissue-library-behavior.component.ts

```diff
@@ -48,4 +51,50 @@ export class TissueLibraryBehaviorComponent {
       }
     });
   }
+
```

Comment:
Move all this logic into the pure component and add an output. Might even be able to programmatically emit a click on a node to trigger navigation through the hraLink